site/pages/api/type-of-consequence.ts

```diff
@@ -0,0 +1,53 @@
+import { NextApiResponse } from "next";
```

Comment:
This file will just need renaming to `type-of-consequence.api.ts`

Reply:
done